syntax = "proto2";
package pb;

message TestMsg
{
    optional int32 id = 1;
    optional string name = 2;
}

message iTestPingShopREQ
{
    optional string ping_msg = 1;
}

message iTestPingActivityMsg
{
    optional string ping_msg = 1;
}

message iTestPingActivityREQ
{
    optional string ping_msg = 1;
}
message iTestPingActivityRSP
{
    optional string pong_msg = 1;
}

message iTestForwardMsg
{
    optional string remote_node_name = 1;
    optional string remote_service_name = 2;
    optional bytes netmsg = 3;
}